pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -204,16 +204,10 @@ let
     inherit commonStructuredConfig structuredExtraConfig extraMakeFlags isZen isHardened isLibre modDirVersion;
     isXen = lib.warn "The isXen attribute is deprecated. All Nixpkgs kernels that support it now have Xen enabled." true;
     passthru = kernel.passthru // (removeAttrs passthru [ "passthru" ]);
-    tests = let
-      overridableKernel = finalKernel // {
-        override = args:
-          lib.warn (
-            "override is stubbed for NixOS kernel tests, not applying changes these arguments: "
```

Comment:
But also, I don't know why this is done, there's no reasoning for it in the original PR

Reply:
https://github.com/NixOS/nixpkgs/pull/121518/files#r663978966

Will take a look tomorrow.